react-ui-kit/src/Inputs/Suggest/components/Menu.tsx

```diff
@@ -1,20 +1,17 @@
-import React from 'react';
+import React, { FC } from 'react';
+import { MenuProps } from 'react-select';
 
-import { MenuListWrapper, MenuWrapper } from './style/MenuStyle';
+import { IInputSuggestOption } from '../index';
```

Comment:
```suggestion
import type { IInputSuggestOption } from '../';
```
Если не указывать `index`,то вроже тоже должно работать